src/modules/integrator-guide/pages/accessing-corteza/ql.adoc

```diff
@@ -100,6 +100,20 @@ Filters for multi-value fields look precisely the same as they would for regular
 | selects = 'a2'
 | The filter returns the records where either one of the multi-value select options equals `a2`.
 
+// [#query-mv-example-6]#<<query-mv-example-6, -> do we need this part?
```

Comment:
Yes we need it and no, there is no example.

Here is the TL;DR of it.
In some scenarios, I want the first table column to act like a title (when you click on it it anchors to it).
This part `[#query-mv-example-6]` defines an anchor (the label can be arbitrary; keep it consistent).
This part `<<query-mv-example-6,Filter for concrete users>>` points to the anchor (the second bit in the <<1, 2>> is the label to use for the link.
The `#` around `<<1, 2>>` are required because reasons (I forgot exactly why but it wouldn't work unless the second part was in some block-ish thing.